pkg/module_manager/module_manager.go

```diff
@@ -158,7 +161,9 @@ func NewModuleManager(ctx context.Context, dirCfg DirectoryConfig, deps *ModuleM
 
 		ValuesValidator: validation.NewValuesValidator(),
 
-		modules:                     new(ModuleSet),
+		modules:           new(ModuleSet),
+		registerModulesGV: regModulesGV,
```

Comment:
Maybe it is better to use `schema.GroupVersionResource` instead of a string?

Reply:
Not actual